src/test/java/com/stripe/util/CaseInsensitiveMapTest.java

```diff
@@ -1,18 +1,16 @@
 package com.stripe.util;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 import java.util.AbstractMap;
 import java.util.Map;
+
+import com.stripe.StripeClient;
```

Comment:
Unused import

Reply:
Thank you for bringing this issue to me will be fixed promptly